Pollo/Views/Cards/CardCell.swift

```diff
@@ -284,6 +278,13 @@ extension CardCell: PollOptionsSectionControllerDelegate {
         return delegate.isConnected
     }
     
+    var maxCellHeight: CGFloat {
+        let cardCellTopHeight = LayoutConstants.hamburgerCardCellHeight + LayoutConstants.questionCellHeight + LayoutConstants.pollMiscellaneousCellHeight
+        + LayoutConstants.separatorLineCardCellHeight
+        let belowCardCellHeight = responsivePadding + (delegate.userRole != .admin ? 0 : questionButtonHeight + questionButtonBottomPadding + timerLabelFontSize + timerLabelBottomPadding)
```

Comment:
can we put `questionButtonHeight + questionButtonBottomPadding + timerLabelFontSize + timerLabelBottomPadding` into a constant explaining what that represents